_pages/participants.md

```diff
@@ -258,31 +254,31 @@ author_profile: false
 * Dr Alex de Crespigny
 * Professor Srinivas Sridhar
 
-## Australia
+## 🇦🇺️ Australia
 * Prof. Roger Evans
 * Dr Dana Kim
 
-## Bosnia And Herzegovina
+## 🇧🇦️ Bosnia And Herzegovina
 * Prof. Sinisa Ristic
 * Prof. Amira Serifovic-Trbalic
 
-## India
+## 🇮🇳️ India
 * Dr Nitin P Ghonge
```

Comment:
```suggestion
* Dr. Nitin P Ghonge
```
Can we directly put the dots uniformly after Dr. and Prof.?

Reply:
yes ;-)